src/Resources/config/services.yml

```diff
@@ -27,6 +27,18 @@ services:
         tags:
             - { name: kernel.cache_warmer }
 
+    contao.controller.abstract_csv_import:
+        class: Contao\CoreBundle\Controller\AbstractCsvImportController
+        abstract: true
+        arguments:
+            - "@database_connection"
+            - "@contao.framework"
+            - "@security.token_storage"
+
+    contao.controller.backend_csv_import:
```

Comment:
I don't know if controllers should be private but I guess not. According to the docs ```Simply said: A service can be marked as private if you do not want to access it directly from your code.``` but I am not sure if it relates to the controllers.